Content.Server/GameTicking/GameTicker.cs

```diff
@@ -214,7 +215,35 @@ public void EndRound()
 
             RunLevel = GameRunLevel.PostRound;
 
-            SendServerMessage("The round has ended!");
+            //Tell every client the round has ended.
+            var roundEndMessage = _netManager.CreateNetMessage<MsgRoundEndMessage>();
+            roundEndMessage.GamemodeTitle = MakeGamePreset().ModeTitle;
+
+            //Get the timespan of the round.
+            var gameTime = IoCManager.Resolve<IGameTiming>();
+            roundEndMessage.RoundDuration = gameTime.RealTime;
+
+            //Generate a list of basic player info to display in the end round summary.
+            var listOfPlayerInfo = new List<RoundEndPlayerInfo>();
+            foreach(var ply in _playerManager.GetAllPlayers().OrderBy(p => p.Name))
+            {
+                if (ply == null) continue;
```

Comment:
It really should not be possible to ever have a player in here be null so you should just remove this check.